server/src/main.rs

```diff
@@ -21,7 +23,11 @@ fn main() {
         Ok(conn) => conn,
         Err(error) => panic!("Could not connect to database: \n\t{}", error)
     };
-    println!("Successfully Connected to db: {}", conn.is_active());
+    unsafe {
+        DB = Some(conn);
+    }
 
+    migrate(); // create all the tables.
```

Comment:
Isn't there a migration framework that we can use? Running migrations every time we start the application can bite us (_e.g._ moving data from one table to another).

Reply:
J'utilise unsafe pour avoir une variable globale

https://doc.rust-lang.org/book/ch19-01-unsafe-rust.html#accessing-or-modifying-a-mutable-static-variable

Mais j'ai mis a jour le code pour réduire au maximum les block unsafe !

Reply:
Sinon Je ne sais pas si c'est nécessaire pour notre application d'utiliser un framework de migrations. C'est peux-être de l'over-engineering... La c'est juste pour créer nos tables et si on fait un gros changement au pire on drop la bdd :)